esp-hal/src/uart.rs

```diff
@@ -278,15 +278,15 @@ pub enum Error {
     /// This error occurs when the received data does not conform to the
     /// expected UART frame format.
     #[allow(clippy::enum_variant_names, reason = "Frame error is a common term")]
```

Comment:
can we remove the clippy-allow now?

Reply:
Tried it locally, seems to be working, thanks!